data/sheragi/archaeology missions.txt

```diff
@@ -332,11 +332,23 @@ outfit "Timer Submunition"
 		"hull damage" .15
 		"no damage scaling"
 
+event "remove pirates from Ildaria"
+	system Ildaria
+		remove fleet "Small Southern Pirates"
+		remove fleet "Large Southern Pirates"
+
+event "readd pirates to Ildaria"
+	system Ildaria
+		add fleet "Small Southern Pirates" 1000
+		add fleet "Large Southern Pirates" 2000
```

Comment:
The event "initial deployment 2" increases the periods of these fleets to 4000 and 7000, respectively. Applying this event after that will result in the pirate frequency in this system being almost four times what it is supposed to be.

Reply:
```suggestion
event "readd pirates to Ildaria"
	system Ildaria
		add fleet "Small Southern Pirates" 4000
		add fleet "Large Southern Pirates" 7000
```
good catch, and there's no way you get this before said event so no need to have 2 different events

Reply:
I did there might be riots before Geminus was bombed

Reply:
I think that's an edge case? I don't think it's worth it to have two events
We're talking about doing TMBR 1,2 and 3, meaning you'd need "Deep Archaeology 5" and going to hai space. Wouldn't happen except if you wanted it to

The one case where I could see adding a different event be worth it is if we make the start of the game happen earlier somehow
A second event wouldnt cost that much to add either

Reply:
The on complete block has a conversation, meaning the conversation can use a branch depending on whether or not initial deployment 2 has happened to determine whether or not to use an action to set one event or the other.

Reply:
I mean dont worry I know pretty much everything about data stuff after having done 10k lines of missions, it isn't about how to do it just whether or not it was worth it
I'll do it quickly and we'll see what warp thinks. It would be more robut tbf

Reply:
Done, but just a word from the devs and I'll revert it as I dont think we should be this precise with spawnrates system

Reply:
Yeah, devs definitely override me.